corehq/apps/users/bulk_download.py

```diff
@@ -212,13 +214,27 @@ def parse_mobile_users(domain, user_filters, task=None, total_count=None):
     current_user_downloaded_count = 0
     for current_domain in domains_list:
         location_cache = LocationIdToSiteCodeCache(current_domain)
+        if EnterpriseMobileWorkerSettings.is_domain_using_custom_deactivation(domain):
+            deactivation_triggers = dict(
+                (f.user_id, f.deactivate_after.strftime('%m-%Y'))
+                for f in DeactivateMobileWorkerTrigger.objects.filter(domain=domain)
+            )
```

Comment:
nit: A dict comprehension would work well here
```suggestion
            deactivation_triggers = {
                f.user_id: f.deactivate_after.strftime('%m-%Y')
                for f in DeactivateMobileWorkerTrigger.objects.filter(domain=domain)
            }
```